MerchantSDK/GiniMerchantSDKExample/GiniMerchantSDKExample/Sources/InvoicesList/OrderDetailViewController.swift

```diff
@@ -303,7 +309,7 @@ extension OrderDetailViewController: PaymentProvidersBottomViewProtocol {
                 self?.showErrorsIfAny()
             } else if let paymentRequestID {
                 self?.dismiss(animated: true, completion: {
-                    self?.paymentComponentsController.openPaymentProviderApp(requestId: paymentRequestID, universalLink: self?.paymentComponentsController.selectedPaymentProvider?.universalLinkIOS ?? "")
+                    self?.paymentComponentsController.openPaymentProviderApp(requestID: paymentRequestID, universalLink: self?.paymentComponentsController.selectedPaymentProvider?.universalLinkIOS ?? "")
```

Comment:
[Clarification]: we are not consistent with the naming of "ID" or "Id"?
Which version do we use in the public documentation?

Reply:
We use both. We should rename everything in a single format. I'll go for `requestId` and `documentId` format. @zladzeyka do you agree?

Reply:
We have more `Id` styles than `ID` variants. I'll go for the `requestId` and `documentId` format